exercises/saddle-points/canonical-data.json

```diff
@@ -0,0 +1,87 @@
+{
+  "exercise": "saddle-points",
+  "version": "1.0.0",
+  "comments": [
+    "Matrix rows and columns are 0-indexed."
+  ],
+  "cases": [
+    {
+      "description": "Can identify single saddle point",
+      "comments": [
+        "This is the README example."
+      ],
+      "property": "saddlePoints",
+      "input": [
+        [9, 8, 7],
+        [5, 3, 2],
+        [6, 6, 7]
+      ],
+      "expected": [
+        {
+          "row": 1,
+          "column": 0
+        }
+      ]
+    },
+    {
+      "description": "Can identify that empty matrix has no saddle points",
+      "property": "saddlePoints",
+      "input": [
+        []
+      ],
+      "expected": []
```

Comment:
@ErikSchierboom forgive me, but I don't see where the line you quoted exists. The description for this exercise reads as follows:

> A matrix may have zero or more saddle points.
> 
> Your code should be able to provide the (possibly empty) list of all the
saddle points for any given matrix.

Can you please point me to the source of the line you quoted? Thanks!